docs/arch/scopes.rst

```diff
@@ -10,13 +10,17 @@ Each token has zero or more scopes associated with it.
 Protected applications indicate which scopes are required to access that application via the scope parameter on the ``/auth`` route.
 When users create their own tokens, they can choose which scopes to delegate to the user token to restrict its power.
 
-Scopes are derived from the user's group membership as determined by their OpenID Connect claims (if OpenID Connect authentication is used), their GitHub team memberships (if GitHub authentication is used), or their LDAP data (if LDAP is configured).
+Scopes are derived from the user's group membership as determined by their OpenID Connect claims (if OpenID Connect authentication is used), their GitHub team memberships (if GitHub authentication is used), or their LDAP group membership (if LDAP is configured).
 The mapping of groups to scopes is controlled by the ``config.groupMappings`` Helm chart value.
 See :ref:`scopes` for more information.
 
-The list of supported scopes and their human-readable descriptions are configured in the ``known_scopes`` configuration setting.
+The list of supported scopes and their human-readable descriptions are configured in the ``config.knownScopes`` configuration setting.
```

Comment:
```suggestion
The list of supported scopes and their human-readable descriptions are configured in the ``config.knownScopes`` Helm configuration setting.
```

(Just a note I was initially wondering about the context of this configuration. It's a Helm value, of course, but someone looking at the architectural docs might also be thinking in terms of the in-python configuration.)